BaseClasses.py

```diff
@@ -803,25 +808,36 @@ def remove(self, item: Item):
             self.stale[item.player] = True
 
 
+class EntranceType(IntEnum):
+    ONE_WAY = 1
+    TWO_WAY = 2
+
+
 class Entrance:
+
     access_rule: Callable[[CollectionState], bool] = staticmethod(lambda state: True)
     hide_path: bool = False
     player: int
     name: str
     parent_region: Optional[Region]
     connected_region: Optional[Region] = None
+    er_group: str
+    er_type: EntranceType
     # LttP specific, TODO: should make a LttPEntrance
     addresses = None
     target = None
 
-    def __init__(self, player: int, name: str = '', parent: Region = None):
+    def __init__(self, player: int, name: str = "", parent: Region = None,
+                 er_group: str = "Default", er_type: EntranceType = EntranceType.ONE_WAY):
         self.name = name
         self.parent_region = parent
         self.player = player
+        self.er_group = er_group
+        self.er_type = er_type
```

Comment:
I did think about it but was weighing against being clear these fields (which are optional) are only relevant to ER and I didn't really want to spell out entrance_rando_type. But this is a good point, I will think about it some more

Reply:
I've renamed these to `randomization_group` and `randomization_type` - this makes it clear they are only for randomization, including "entrance" in the name is redundant because they are members of entrances.